radio/src/CMakeLists.txt

```diff
@@ -979,14 +979,15 @@ if(ARCH STREQUAL ARM)
   set(CMAKE_ASM_COMPILE_OBJECT "<CMAKE_ASM_COMPILER> -o <OBJECT> <SOURCE>")
 
   set(COMMON_FLAGS "-mcpu=${MCU} -mthumb -fomit-frame-pointer -fverbose-asm -Wa,-ahlms=firmware.lst -O${OPT} -gdwarf-2 -DHSE_VALUE=${HSE_VALUE} -fno-exceptions -fdata-sections -ffunction-sections ${WARNING_FLAGS}")
+  set(COMMON_CPP_FLAGS "${COMMON_FLAGS} -std=gnu++11")
```

Comment:
Seem like ubuntu has the required version only since 16.04 (not yet released iirc): http://packages.ubuntu.com/search?keywords=cmake&searchon=names&suite=all&section=all

Maybe add the c++11 flags to CMAKE_CXX_FLAGS depending on cmake version for now and later (in a year or two) force cmake 3.1?